src/main/java/org/mule/tools/devkit/sonar/checks/maven/DevkitLatestVersionCheck.java

```diff
@@ -0,0 +1,47 @@
+package org.mule.tools.devkit.sonar.checks.maven;
+
+import com.google.common.collect.Lists;
+import org.apache.maven.project.MavenProject;
+import org.mule.tools.devkit.sonar.checks.ConnectorIssue;
+import org.mule.tools.devkit.sonar.utils.PomUtils;
+import org.mule.tools.devkit.sonar.utils.VersionUtils;
+import org.sonar.check.Priority;
+import org.sonar.check.Rule;
+
+import java.util.List;
+
+@Rule(key = DevkitLatestVersionCheck.KEY, name = "Devkit version should be the latest stable release", description = "This rule checks whether the current connector is using the latest stable release version of Devkit", priority = Priority.MAJOR, tags = {"connector-certification"})
+public class DevkitLatestVersionCheck implements MavenCheck {
+
+    public static final String KEY = "devkit-latest-version";
+    private static final int OUTDATED_VERSION = 1;
+    private VersionUtils latestVersion;
+    private VersionUtils currentVersion;
+
+    @Override
+    public Iterable<ConnectorIssue> analyze(MavenProject mavenProject) {
+        final List<ConnectorIssue> issues = Lists.newArrayList();
+        String devkitVersion = mavenProject.getModel().getParent().getVersion();
+        String issueMessage;
+        latestVersion = PomUtils.setLatestVersion();
+        if (devkitVersion.indexOf('-') != -1) {
+            issueMessage = String.format("Current connector Devkit version '%s' is not the last stable version. If feasible, use the latest stable one '%s'.", devkitVersion, latestVersion);
+            issues.add(new ConnectorIssue(KEY, issueMessage));
+        } else {
+            currentVersion = new VersionUtils(devkitVersion);
+            if (latestVersion.compareTo(currentVersion) == OUTDATED_VERSION) {
```

Comment:
I would simplify these conditional in the following manner. Let me know if you agree or  if I missed some use case:

```public Iterable<ConnectorIssue> analyze(MavenProject mavenProject) {
        final List<ConnectorIssue> issues = Lists.newArrayList();
        String devkitVersion = mavenProject.getModel().getParent().getVersion();
        VersionUtils latestVersion = PomUtils.getLatestDevKitVersion();
        if (isRevision(devkitVersion) || (!isRevision(devkitVersion) && latestVersion.compareTo(PomUtils.getCurrentDevKitVersion(devkitVersion)) == OUTDATED_VERSION)){
            issues.add(new ConnectorIssue(KEY, String.format("Current connector Devkit version [%s] is not the latest stable version. If feasible, use the latest DevKit [%s].", devkitVersion, latestVersion)));
        }
        return issues;
    }
        }
...

private boolean isRevision(String devkitVersion){
        return devkitVersion.indexOf('-') != -1;
    }